oak/server/rust/oak_runtime/src/config.rs

```diff
@@ -101,6 +101,18 @@ pub fn from_protobuf(
                     })?,
                     tls_identity: Identity::from_pem(grpc_tls_certificate, grpc_tls_private_key),
                 },
+                Some(ConfigType::GrpcClientConfig(GrpcClientConfiguration {
+                    address,
+                    domain_name,
+                    root_tls_certificate,
+                })) => node::Configuration::GrpcClientNode {
+                    address: address.parse().map_err(|error| {
+                        error!("Incorrect gRPC server address: {:?}", error);
+                        OakStatus::ErrInvalidArgs
+                    })?,
+                    domain_name: domain_name.to_string(),
+                    root_tls_certificate: Certificate::from_pem(root_tls_certificate),
```

Comment:
The lack of a `?` here looks suspicious &ndash; what happens if someone feeds in an invalid PEM file?

Reply:
OK, so any parse errors from the two layers of encoding (PEM for ASCII->bytes, ASN.1 DER for bytes->certificate data structure) or validity errors (invalid dates, unsupported key formats, etc) are deferred until then&hellip;

Reply:
Added the `load_certificate` function that checks certificates.